C-Plus-Plus/cryptography/VernamCipher.cpp

```diff
@@ -0,0 +1,67 @@
+//Vernam Cipher
+
+// Vernam Cipher is a symmetric key cryptographic algorithm and is one of the transposition technique that converts plain text into ciphertext.
+// In this algorithm the size of the key and plain text must be same.
+
+
+#include<bits/stdc++.h>
+using namespace std;
+
+string encrypt(string plain,string key){
+	string enc="";
+	int n=key.length();
+  for(int i=0;i<n;i++){
+		int y = plain[i]+key[i]-130;
+		if(y>=26) y = y-26;
+		enc+=char(y+65);
+	}
+	return enc;
+}
+
+string decrypt(string enc,string key){
+	string dec="";
+	int n=enc.length();
+	for(int i=0;i<n;i++){
+		if(enc[i]>=key[i]){
+			dec+=char(enc[i]-key[i]+65);
+		}
+		else{
+			dec+=char(enc[i]-key[i]+91);  // enc[i]-key[i]+26+65
```

Comment:
Use comments above the Code and not beside them. Don't keep them ambiguous just for the purpose of debugging.